gui/public/i18n/es-419/translation.ftl

```diff
@@ -438,9 +438,9 @@ onboarding-assign_trackers-description = Debes escoger dónde van los sensores.
 # $assigned (Number) - Trackers that have been assigned a body part
 # $trackers (Number) - Trackers connected to the server
 onboarding-assign_trackers-assigned = { $assigned } of { $trackers ->
```

Comment:
Mira desde el ``=``, pone la variable $assigned y despues hay un ``of`` que es texto.
Osea justo ahora se ve asi la traduccion: ``1 of 1 sensor asignado``